components/docs-chef-io/content/automate/architectural_overview_os.md

```diff
@@ -0,0 +1,79 @@
++++
```

Comment:
We should update the existing architecture page with OpenSearch updates. No new page for architecture.